pkg/cliplugins/workspace/plugin/kubeconfig_test.go

```diff
@@ -753,7 +753,7 @@ func TestUse(t *testing.T) {
 				AuthInfos: map[string]*clientcmdapi.AuthInfo{"test": {Token: "test"}},
 			},
 			existingObjects: map[logicalcluster.Name][]string{
-				core.RootCluster: {"~"},
+				core.RootCluster: {"~", homeWorkspaceLogicalCluster.String()},
```

Comment:
Not sure I understand why this addition is necessary.
Afaict, it the failing case would be better tested if we distinguish between `homeWorkspaceLogicalCluster` *name* and `homeWorkspaceLogicalCluster` hierarchical path. Afaik for now we use the same value (path everywhere). But the failing test was precisely related to the fact that those 2 values are different.

Reply:
this is not necessary, removed.

Reply:
> Afaict, it the failing case would be better tested if we distinguish between homeWorkspaceLogicalCluster name and homeWorkspaceLogicalCluster hierarchical path. Afaik for now we use the same value (path everywhere). But the failing test was precisely related to the fact that those 2 values are different.

what about this part of my comment ?